### PR TITLE
Fixed SlimyTreeTaps rubber conflict with LX Rubber and added the Wrench

### PR DESCRIPTION
**Pull request by NCBPFluffyBear**
## Short Description
Fixed SlimyTreeTaps rubber conflict with LX Rubber and added the Wrench

## Additions/Changes/Removals
SlimyTreeTaps: Added to POM, LX rubber and Rubber Synthesizer are now disabled when SlimyTreeTaps exists

Wrench: New tool that instantly breaks capacitors and cargo nodes. Drops the inventories of cargo inputs and advanced outputs.

## Related Issues
N?A

## Checklist

- [x] I have fully tested the proposed changes and promise that they will not break everything into chaos.
- [x] I have also tested the proposed changes in combination with base Slimefun and made sure nothing breaks/unexpected happens.
- [x] I followed the existing code standards and didn't mess up the formatting.
- [x] I did my best to add documentation to any public classes or methods I added which may not be obvious to maintainers.
- [x] I have added `Nonnull` and `Nullable` annotations to my methods to indicate their behaviour for null values

### CODE REVIEW

**Comment by NCBPFluffyBear** [comment]
Actually, this also fixes https://github.com/J3fftw1/LiteXpansion/issues/45

File: src/main/java/dev/j3fftw/litexpansion/ItemSetup.java [hunk 1]
```diff
@@ -21,6 +22,8 @@
 
 import javax.annotation.Nonnull;
 
+import static org.bukkit.Bukkit.getServer;
```
Comment (WalshyDev) [inline src/main/java/dev/j3fftw/litexpansion/ItemSetup.java:25]: Why the static import here...

**Comment by NCBPFluffyBear** [comment]
Ima close this so i can switch branches

File: src/main/java/dev/j3fftw/litexpansion/items/Wrench.java [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+
+import static org.bukkit.Bukkit.getLogger;
+
+public class Wrench extends SimpleSlimefunItem<ItemUseHandler> implements Listener {
+
+    protected static final int[] cargoSlots = { 19, 20, 21, 28, 29, 30, 37, 38, 39 };
+    protected static final String[] wrenchableBlockIds = { SlimefunItems.CARGO_INPUT_NODE.getItemId(), SlimefunItems.CARGO_OUTPUT_NODE.getItemId(), SlimefunItems.CARGO_OUTPUT_NODE_2.getItemId(), SlimefunItems.CARGO_CONNECTOR_NODE.getItemId(), SlimefunItems.SMALL_CAPACITOR.getItemId(), SlimefunItems.MEDIUM_CAPACITOR.getItemId(), SlimefunItems.LARGE_CAPACITOR.getItemId(), SlimefunItems.BIG_CAPACITOR.getItemId(), SlimefunItems.CARBONADO_EDGED_CAPACITOR.getItemId(), SlimefunItems.TRASH_CAN.getItemId() };
```
Comment (WalshyDev) [inline src/main/java/dev/j3fftw/litexpansion/items/Wrench.java:38]: This kind is ridiculously long, formatting is good. Also why are these protected.

File: src/main/java/dev/j3fftw/litexpansion/items/Wrench.java [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+    public ItemUseHandler getItemHandler() {
+        return e -> e.setUseBlock(Event.Result.DENY);
+    }
+
+    @EventHandler
+    public void onWrenchInteract(PlayerInteractEvent e) {
+        Player p = e.getPlayer();
+
+        if (e.getAction() == Action.RIGHT_CLICK_BLOCK) {
+            if (SlimefunUtils.isItemSimilar(e.getItem(), Items.WRENCH, true, false)) {
```
Comment (WalshyDev) [inline src/main/java/dev/j3fftw/litexpansion/items/Wrench.java:60]: Don't make this an inner if

**Review by WalshyDev — COMMENTED** [review]
Why have you made this only work on cargo and caps? It should be on any machine. If we are gonna implement it should be implemented fully.

File: src/main/java/dev/j3fftw/litexpansion/ItemSetup.java (hunk 1 above)
Comment (NCBPFluffyBear) [reply src/main/java/dev/j3fftw/litexpansion/ItemSetup.java:25]: Im not really sure how these work, It auto imported when I used getServer, and getting rid of static breaks stuff... Could u explain pls

File: src/main/java/dev/j3fftw/litexpansion/items/Wrench.java (hunk 2 above)
Comment (NCBPFluffyBear) [reply src/main/java/dev/j3fftw/litexpansion/items/Wrench.java:38]: Is there a good way to cut down the size

File: src/main/java/dev/j3fftw/litexpansion/ItemSetup.java (hunk 1 above)
Comment (WalshyDev) [reply src/main/java/dev/j3fftw/litexpansion/ItemSetup.java:25]: Bukkit.getServer

You're calling a method from the Bukkit class.

File: src/main/java/dev/j3fftw/litexpansion/items/Wrench.java (hunk 2 above)
Comment (WalshyDev) [reply src/main/java/dev/j3fftw/litexpansion/items/Wrench.java:38]: Just use multiple lines.

**Pull request closed without merging**